configure.sh

```diff
@@ -75,6 +75,26 @@ configure_app_serverinfo() {
 	ooc config:app:set serverinfo token --value "${NC_APP_SERVERINFO_TOKEN}"
 }
 
+configure_app_richdocuments() {
+	if ! [ "${COLLABORA_HOST}" ] ; then
+		fail Collabora host is not set
+	fi
+
+	if ! [ "${COLLABORA_EDIT_GROUPS}" ] ; then
+		fail Collabora edit groups are not set
+	fi
+
+	ooc app:enable richdocuments
+	ooc config:app:set richdocuments wopi_url --value=${COLLABORA_HOST}
+	ooc config:app:set richdocuments public_wopi_url --value=${COLLABORA_HOST}
```

Comment:
Double quote to prevent globbing and word splitting.
See SC2086. https://github.com/koalaman/shellcheck/wiki/SC2086

```suggestion
	ooc config:app:set richdocuments wopi_url --value="${COLLABORA_HOST}"
	ooc config:app:set richdocuments public_wopi_url --value="${COLLABORA_HOST}"
```